### ISSUE TITLE
ASP.NET Core Web Application - Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal.ConnectionResetException

### ISSUE THREAD

**Issue by lavinianastase** [aspnet/Docs#4743]
On Visual Studio 2017, I created an ASP.NET Core Web Application with the following options: 
![setting](https://user-images.githubusercontent.com/5435139/32547764-3e346b06-c48c-11e7-86cf-9f666488ef16.PNG)

I haven't changed anything else. When I run the project on Debug with Any CPU option (other configurations behave the same), I get the following error:

![capture](https://user-images.githubusercontent.com/5435139/32547772-45d9e5a2-c48c-11e7-9deb-275f68e99ae7.PNG)

Additional error:
![error](https://user-images.githubusercontent.com/5435139/32547824-6ef01ec0-c48c-11e7-968c-0fd509f57078.PNG)

I reinstalled .NET core framework, no luck.

Any ideas? Thank you in advance.

**Comment by Rick-Anderson** [comment]
@lavinianastase  I've never seen this error and I cannot reproduce it. Can you try:

* Rebooting
* Creating a Razor Pages web app

Do you get this error when you run non-Debug?

**Comment by halter73** [comment]
@lavinianastase Do you see any issues if you don't break on first chance exceptions?

The ConnectionResetException is an IOException that can be thrown whenever a client disconnects ungracefully. There is a pretty good chance that if you don't break when the exception is first thrown, you wouldn't notice it except for perhaps in a debug-level log.

**Comment by lavinianastase** [comment]
Hi,
Thank you for your answer. To answer your questions above, I got the same error when running non-debug and the application didn't start even though I didn't break on first chance exceptions.
The issue is no longer reproducible and I took the following steps:
- uninstall and reinstall Visual Studio 2017
- reinstall IIS
- reboot.
It is still a mystery the root cause of this issue, since I didn't actually do any modifications before this.
